src/eterna/ui/Toolbar.ts

```diff
@@ -267,12 +354,71 @@ export default class Toolbar extends ContainerObject {
 
         // LOWER TOOLBAR (palette, zoom, settings, etc)
         this.lowerToolbarLayout = new HLayoutContainer(0, VAlign.BOTTOM);
+        this.backgroundContainer = new Container();
+
+        this.backgroundContainerBackgroundContainer = new Graphics()
+            .beginFill(0x21508C, 1)
+            .drawRoundedRect(0, 0, 666, 223, 7)
+            .endFill();
+        this.backgroundContainerBackgroundContainer.visible = false;
+
+        this.lowerToolbarContainer = new VLayoutContainer();
+        this.backgroundContainer.addChild(this.backgroundContainerBackgroundContainer);
+        this.backgroundContainer.addChild(this.lowerToolbarContainer);
+        this.lowerToolbarLayout.addChild(this.backgroundContainer);
+        this.top = new HLayoutContainer();
+        this.middle = new Container();
+        const background = new Graphics()
+            .beginFill(0x043468)
+            .drawRoundedRect(0, 0, 600, 51, 7)
+            .endFill();
+        this.middle.addChild(background);
+        this.middle.visible = false;
+        this.lowerToolbarContainer.addChild(this.top);
+        this.lowerToolbarContainer.addVSpacer(14);
+        this.lowerToolbarContainer.addChild(this.middle);
+        // eslint-disable-next-line max-len
+        this.text = new Text('Drag an icon to the right or left above to replace the existing tool, or tap an icon to use it.', {
+            fontSize: 12,
+            fontFamily: Fonts.STDFONT,
+            fill: 0xABC9D8,
+            fontWeight: FontWeight.MEDIUM
+        });
+        this.text.visible = false;
+        this.lowerToolbarContainer.addVSpacer(14);
+        this.lowerToolbarContainer.addChild(this.text);
         this.scrollContainerContainer = new HLayoutContainer(0, VAlign.BOTTOM);
         Assert.assertIsDefined(Flashbang.stageHeight);
         this.scrollContainer = new ScrollContainer(Flashbang.stageWidth, Flashbang.stageHeight);
         this._content.addChild(this.scrollContainerContainer);
         this.scrollContainer.container.addChild(this.lowerToolbarLayout);
 
+        DisplayUtil.positionRelative(
+            this.lowerToolbarContainer, HAlign.CENTER, VAlign.TOP,
+            this.backgroundContainer, HAlign.CENTER, VAlign.TOP,
+            0, 20
+        );
+
+        this._isDragging = false;
+
+        this.backgroundContainer.interactive = true;
+
+        this.backgroundContainer.on('pointerup', (e: InteractionEvent) => {
```

Comment:
Use a ContainerObject instead of a Container so that you can use the utilities like `.pointerDown.connect(() => {})`